### PR TITLE
Correctly used markdown markups (deleted HTML escaped as &gt; or &lt;)

### PR DESCRIPTION
**Pull request by adriaroms**
Simply: Correctly used markdown markups (deleted HTML escaped as &gt; or &lt;), not visualized.

Adrià.

### CODE REVIEW

File: i18n/en-US/generic/Props.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ export default {
   fixed: 'Position the element fixed',
   hideDetails: 'Hides hint, validation errors',
   hint: 'Hint text',
-  href: 'Will designate the component tag to <code>&lt;a&gt;</code>',
+  href: 'Will designate the component tag to <code>`<a>`</code>',
```
Comment (johnleider) [inline i18n/en-US/generic/Props.js]: remove code

File: i18n/en-US/generic/Props.js [hunk 2]
```diff
@@ -73,7 +73,7 @@ export default {
   target: 'Specify the target attribute, only works with anchor tag.',
   textColor: 'Applies a specified color to the control text',
   tile: 'Removes border radius',
-  to: 'Will designate the component tag to <code>&lt;router-link&gt;</code>',
+  to: 'Will designate the component tag to <code>`<router-link>`</code>',
```
Comment (johnleider) [inline i18n/en-US/generic/Props.js]: remove code

File: i18n/en-US/getting-started/FrequentlyAskedQuestions.js [hunk 3]
```diff
@@ -45,7 +45,7 @@ export default {
     },
     {
       q: `My page on mobile is not responsive`,
-      a: `Add the <code>&lt;meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no, minimal-ui"&gt;</code> to the <code>&lt;head&gt;</code> section in your index.html.`
+      a: 'Add the <code>`<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no, minimal-ui">`</code> to the <code>`<head>`</code> section in your index.html.'
```
Comment (johnleider) [inline i18n/en-US/getting-started/FrequentlyAskedQuestions.js]: remove code tag

File: i18n/en-US/guides/ALaCarte.js [hunk 4]
```diff
@@ -1,11 +1,11 @@
 export default {
   header: 'Overview',
```
Comment (johnleider) [inline i18n/en-US/guides/ALaCarte.js]: remove these changes, I've already started changing this

File: i18n/en-US/guides/ServerSideRendering.js [hunk 5]
```diff
@@ -14,7 +14,7 @@ export default {
...
-  routingText2: 'These routes can be accessed by creating a link to the specified path, or by using Vue Router\'s <code>&lt;router-link&gt;</code> component. For more information, review the official Vue Router <a href="https://router.vuejs.org/en/" target="_blank" rel="noopener">documentation</a>. The default function assumes you are placing views in the <kbd>/pages</kbd> folder containing **View** on the end of the file. Example <code>HomeView.vue</code>.',
+  routingText2: 'These routes can be accessed by creating a link to the specified path, or by using Vue Router\'s <code>`<router-link>`</code> component. For more information, review the official Vue Router <a href="https://router.vuejs.org/en/" target="_blank" rel="noopener">documentation</a>. The default function assumes you are placing views in the <kbd>/pages</kbd> folder containing **View** on the end of the file. Example <code>HomeView.vue</code>.',
```
Comment (johnleider) [inline i18n/en-US/guides/ServerSideRendering.js]: remove code tag

Comment (johnleider) [inline i18n/en-US/guides/ServerSideRendering.js]: change anchor tag to use markdown link

File: i18n/en-US/style/Colors.js [hunk 6]
```diff
@@ -2,7 +2,7 @@ export default {
...
   classesHeader: 'Classes',
-  classesText: `You can change a background or text color by adding a class to the element. For background, use the name, for example, <code>&lt;div class="red"&gt;</code>. To use a shade, just add the modifying type, such as, **darken-3**. Text is very similar, adding a modification class, **red--text**, you can change the color of text, or change the type with adding **text--lighten-2**. These colors are also available as javascript variables, just import colors from <code>vuetify/es5/util/colors</code>`,
+  classesText: 'You can change a background or text color by adding a class to the element. For background, use the name, for example, <code>`<div class="red">`</code>. To use a shade, just add the modifying type, such as, **darken-3**. Text is very similar, adding a modification class, **red--text**, you can change the color of text, or change the type with adding **text--lighten-2**. These colors are also available as javascript variables, just import colors from <code>vuetify/es5/util/colors</code>',
```
Comment (johnleider) [inline i18n/en-US/style/Colors.js]: remove code tag

File: i18n/en-US/style/Content.js [hunk 7]
```diff
@@ -4,9 +4,9 @@ export default {
   blockquoteHeader: 'Block quote',
   paragraphsHeader: 'Paragraphs',
   codeHeader: 'Code',
-  codeText: 'Example of an inline <code>&lt;code&gt;</code> element.',
+  codeText: 'Example of an inline <code>`<code>`</code> element.',
```
Comment (johnleider) [inline i18n/en-US/style/Content.js]: remove code tags surrounding the code tag

File: i18n/en-US/guides/ALaCarte.js [hunk 8]
```diff
@@ -1,11 +1,11 @@
...
-  applicationText2: 'For example the code below tells the application that you use a `<v-app>`, `<v-avigation-drawer>`, `<v-footer>` and `<v-toolbar>`. Once defined, the markup can be used in any .vue file.',
+  applicationText1: 'Navigate to the **src** folder and open up ```main.js```. We see that in the template come some components already configured to load.',
+  applicationText2: 'For example the code below tells the application that you use a ```<v-app>```, ```<v-avigation-drawer>```, ```<v-footer>``` and ```<v-toolbar>```. Once defined, the markup can be used in any .vue file.',
   componentNameListHeader: 'UI Component Name List',
-  componentNameListText1: 'We must be aware that components such as `VLayout` or `VFlex` are not defined as such, for organizational reasons are included within `VGrid`, to know what is the name of a component to import can review the following section',
+  componentNameListText1: 'We must be aware that components such as ```VLayout``` or ```VFlex``` are not defined as such, for organizational reasons are included within ```VGrid```, to know what is the name of a component to import can review the following section',
```
Comment (KaelWD) [inline i18n/en-US/guides/ALaCarte.js]: Use single backticks

Comment (adriaroms) [reply i18n/en-US/guides/ALaCarte.js]: @KaelWD John: "remove these changes, I've already started changing this" https://github.com/vuetifyjs/vuetifyjs.com/pull/109#discussion_r155421448

**Pull request merged**